warehouse/models/mart/gtfs/fct_hourly_rt_feed_files_success.sql

```diff
@@ -0,0 +1,154 @@
+{{ config(materialized='incremental', unique_key = 'key') }}
+
+-- BigQuery does not do partition elimination when using a subquery: https://stackoverflow.com/questions/54135893/using-subquery-for-partitiontime-in-bigquery-does-not-limit-cost
+-- save max date in a variable instead so it can be referenced in incremental logic and still use partition elimination
+{% if is_incremental() %}
+    {% set dates = dbt_utils.get_column_values(table=this, column='dt', order_by = 'dt DESC', max_records = 1) %}
+    {% set max_date = dates[0] %}
+{% endif %}
+
+WITH
+
+int_gtfs_rt__daily_url_index AS (
+    SELECT *
+    FROM {{ ref('int_gtfs_rt__daily_url_index') }}
+),
+
+int_gtfs_rt__unioned_parse_outcomes AS (
+    SELECT *
+    FROM {{ ref('int_gtfs_rt__unioned_parse_outcomes') }}
+),
+
+parse_outcomes AS (
+    SELECT *
+    FROM int_gtfs_rt__unioned_parse_outcomes
+    {% if is_incremental() %}
+    WHERE dt >= DATE '{{ max_date }}'
+    {% else %}
+        WHERE dt >= DATE_SUB(CURRENT_DATE(), INTERVAL 1 MONTH)
+    {% endif %}
+),
+
+daily_tots AS (
+
+    SELECT
+
+        dt,
+        base64_url,
+        feed_type,
+        COUNT(*) AS file_count_day,
+        SUM(
+            CASE parse_success WHEN TRUE THEN 1 ELSE 0 END
+        ) AS success_file_count_day,
+        (
+            SUM(CASE parse_success WHEN TRUE THEN 1 ELSE 0 END) / COUNT(*)
+        ) * 100 AS pct_success_file_count_day
+
+    FROM parse_outcomes
+    GROUP BY
+        dt,
+        base64_url,
+        feed_type
+
+),
+
+hourly_tots AS (
+
+    SELECT
+
+        dt,
+        base64_url,
+        feed_type,
+        EXTRACT(
+            HOUR FROM hour AT TIME ZONE "America/Los_Angeles"
+        ) AS download_hour,
+        SUM(
+            CASE parse_success WHEN TRUE THEN 1 ELSE 0 END
+        ) AS success_file_count_hour,
+        COUNT(*) AS file_count_hour
+
+    FROM parse_outcomes
+    GROUP BY
+        dt,
+        download_hour,
+        base64_url,
+        feed_type
+
+),
+
+pivot_hourly_tots AS (
+
+    SELECT *
+    FROM
+        (SELECT
+
+            dt,
+            download_hour,
+            base64_url,
+            feed_type,
+            (success_file_count_hour / file_count_hour) * 100 AS pct_success_hour
+
+            FROM hourly_tots)
+    PIVOT(
+        SUM(pct_success_hour)
```

Comment:
Just to confirm, we should not actually be summing here, right? There should only be one value per bucket? Just get nervous when I see summing of a percent/proportion, wondering if there is any other way to indicate this

Reply:
Yes you are right - I got tripped up because `PIVOT` requires an aggregation and I couldn't figure out how to get around it, so I just added the SUM() for the time being because I thought it would allow it to run and wouldn't change the results (and meant to note it in the PR). I just changed it to MAX() instead, which is less scary I think, but maybe we can talk about alternatives this afternoon. I bet this CTE and the one before it can be completely rethought...